test/dynamo/test_triton_kernels.py

```diff
@@ -944,6 +944,37 @@ def add_kernel_out_of_order(
             ["out_ptr"],
         )
 
+    @make_mutation_test
+    def test_out_of_order_kernel_call():
+        @triton.jit
+        def add_kernel_out_of_order_fn1(
```

Comment:
If `fn2` is in the `triton_utils.py`, maybe move this one there, too, for consistency?

Reply:
These are temporary kernels, i wanted to keep them all here but unfortunately triton scoping rules are broken.

https://github.com/openai/triton/issues/3035